What's New: Add info about new behavior when you change the question type (#2544)
surveyjs/survey-creator: docs/Creator-V2-Whats-New.md

<a id="action-buttons"></a> 

### Action buttons rearranged

Previously, action buttons were split between different tabs (Designer, Logic, etc.). Now, they are displayed within the question that you configure. You also can access the actions programmatically.

![Action buttons](images/creator-v2-action-buttons.png)  
_Action buttons_

If you click the button that changes the question type, you will see a drop-down list of all supported types. In Survey Creator V1, this list was limited to the types that were interchangeable with the current element type. With the full list, you can now add multiple default questions to the survey to specify their types afterwards.

![List of supported question types](images/creator-v2-question-types.png)   
_Drop-down list of supported question types_

> If you switch between non-interchangeable question types, the question configuration may be lost. Use Undo and Redo buttons to roll back or reinstate the previous configuration.

Reviewer: The following line of code comeback to the old behavior in V1.
`SurveyCreator.settings.questionConvertMode = 1;`
In this case text can be converted into "comment" only, dropdown into checkbox, radiogroup and so on. The behavior we have in V1.

Author: This information is added.

<a id="action-buttons"></a> 

### Action buttons rearranged

> Previously, action buttons were split between different tabs (Designer, Logic, etc.). Now, they are displayed within the question that you configure. You also can access the actions programmatically.

Reviewer 2: ```suggestion
Action buttons used to be split between different tabs (Designer, Logic, etc.). Now, these actions are displayed within the question that you configure and can also be accessed programmatically.
```

![Action buttons](images/creator-v2-action-buttons.png)  
_Action buttons_

If you click the button that changes the question type, you will see a drop-down list of all supported types. In Survey Creator V1, this list was limited to the types that were interchangeable with the current element type. With the full list, you can now add multiple default questions to the survey to specify their types afterwards.

![List of supported question types](images/creator-v2-question-types.png)   
_Drop-down list of supported question types_

> If you switch between non-interchangeable question types, the question configuration may be lost. Use Undo and Redo buttons to roll back or reinstate the previous configuration. If you want to revert to the old behavior (limit the list to interchangeable types only), use the following code:

Reviewer 2: ```suggestion
If you switch between non-interchangeable question types, the question configuration may be lost. Use Undo and Redo buttons to roll back or reinstate the previous configuration. To revert to old behavior (limit the list to interchangeable types only), use the following code:
```
